paasta_tools/marathon_tools.py

```diff
@@ -1076,3 +1076,23 @@ def get_instances_from_zookeeper(service, instance):
     with ZookeeperPool() as zookeeper_client:
         (instances, _) = zookeeper_client.get('%s/instances' % compose_autoscaling_zookeeper_root(service, instance))
         return int(instances)
+
+
+def is_task_healthy(task, require_all=True, default_healthy=False):
+    """
+    Check that a marathon task is healthy
```

Comment:
Fix the formatting here please - this should be on the same line as the `"""`. Could you also add the `:returns` please